agent/structs/check_definition.go

```diff
@@ -70,6 +71,7 @@ func (c *CheckDefinition) CheckType() *CheckType {
 		GRPCUseTLS:        c.GRPCUseTLS,
 		Header:            c.Header,
 		Method:            c.Method,
+		Body:              c.Body,
```

Comment:
Add this new field to the documentation on checks here - https://github.com/hashicorp/consul/blob/master/website/source/docs/agent/checks.html.md 
and here - https://github.com/hashicorp/consul/blob/master/website/source/api/agent/check.html.md

Reply:
I have added the new field to the documentation.